freeipa-desktop-profile.spec

```diff
@@ -22,10 +22,16 @@ BuildRequires: python3-devel
 BuildRequires: python3-ipaserver >= 4.6.0
 %endif
 
-BuildRequires:  python2-devel
-BuildRequires:  python2-ipaserver >= 4.4.1
 
-Requires:       ipa-server-common >= 4.4.1
+%if 0%{?rhel}
+BuildRequires: python3-devel
+BuildRequires: python3-ipaserver >= 4.6.0
+Requires:      ipa-server-common >= 4.4.1
+%else
+BuildRequires:  python2-devel
+BuildRequires:  python2-ipaserver >= 4.4.0
+Requires:       ipa-server-common >= 4.4.0
+%endif
 
```

Comment:
Also, I thought about it more and I think we can keep >= 4.4.0 universally. It may well fail on really old fedora but RHEL never had 4.4 version without external plugin support, so that should be fine.